recipes/matrix-py3/meta.yaml

```diff
@@ -0,0 +1,76 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "matrix-py3" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/54/c8/ef3647906d37e41fbda34a6ffbe8bfc5633dd44276be2ff7844ee52b4ae9/matrix-py3-0.0.1.tar.gz
```

Comment:
The form mentioned [in the example](https://github.com/chunribu/staged-recipes/blob/master/recipes/example/meta.yaml#L14) is preferred for PyPI packages: